packages/react-property/scripts/build-html.js

```diff
@@ -0,0 +1,119 @@
+const fs = require('fs');
+const path = require('path');
+const DOMProperty = require('react-dom/lib/DOMProperty');
+const HTMLDOMPropertyConfig = require('react-dom/lib/HTMLDOMPropertyConfig');
+const { LIB_DIR, HTML_DIR } = require('./constants');
+
+/**
+ * Creates the DOM property map via injection.
+ *
+ * @see https://github.com/facebook/react/blob/15-stable/src/renderers/dom/shared/DOMProperty.js#L57
+ *
+ * ```js
+ * {
+ *   properties: {
+ *     accept: {
+ *       attributeName: 'accept',
+ *       attributeNamespace: null,
+ *       propertyName: 'accept',
+ *       mutationMethod: null,
+ *       mustUseProperty: false,
+ *       hasBooleanValue: false,
+ *       hasNumericValue: false,
+ *       hasPositiveNumericValue: false,
+ *       hasOverloadedBooleanValue: false
+ *     },
+ *     // ...
+ *   },
+ *   getPossibleStandardName: {
+ *     autofocus: 'autoFocus',
+ *     accept: 'accept',
+ *     acceptcharset: 'acceptCharset',
+ *     'accept-charset': 'acceptCharset',
+ *     // ...
+ *   },
+ *   // ...
+ * }
+ * ```
+ */
+DOMProperty.injection.injectDOMPropertyConfig(HTMLDOMPropertyConfig);
+
+/**
+ * Create output directories (if it does not exist).
+ */
+try {
+  fs.mkdirSync(LIB_DIR);
+} catch (error) {
+  // throw error;
+}
+
+try {
+  fs.mkdirSync(HTML_DIR);
+} catch (error) {
+  // throw error;
+}
+
+/**
+ * HTML DOM property config.
```

Comment:
Maybe we can add extra context to this comment? I wasn't sure what it was for at first. Something along the lines of: "an object containing HTML attributes and how they should be mapped to React props"

Reply:
42999e2

I changed the wording slightly. Let me know if it makes sense.

Reply:
Yeah, I think this makes it clearer.